### PR TITLE
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/87 popu…

### PR DESCRIPTION
**Pull request by Mizzick**
…p layout

### CODE REVIEW

File: Extension/pages/skin/my.css [hunk 1]
```diff
@@ -273,6 +273,7 @@ a.spl-user-add-spn {
 }
 
 .popup-footer span {
+    overflow: hidden;
```
Comment (ameshkov) [inline Extension/pages/skin/my.css]: That's simply not enough, have you tested it?

File: Extension/_locales/en/messages.json [hunk 2]
```diff
@@ -402,7 +402,7 @@
         "message": "Open filtering log..."
     },
     "popup_adguard_other_os": {
-        "message": "Adguard for other platforms:"
+        "message": "Other Adguard versions:"
```
Comment (ameshkov) [inline Extension/_locales/en/messages.json]: 1. Export this to oneskyapp (!)

File: Extension/pages/skin/my.css (hunk 1 above)
Comment (Mizzick) [inline Extension/pages/skin/my.css]: yeap

File: Extension/_locales/en/messages.json (hunk 2 above)
Comment (Mizzick) [inline Extension/_locales/en/messages.json]: exported

File: Extension/pages/skin/my.css (hunk 1 above)
Comment (ameshkov) [inline Extension/pages/skin/my.css]: So what? It just breaks the line, overflow hidden does nothing:
!(http://content.screencast.com/users/ameshkov/folders/Jing/media/26211883-a695-491a-a52b-2a843363589b/00000913.png)

File: Extension/_locales/en/messages.json (hunk 2 above)
Comment (ameshkov) [inline Extension/_locales/en/messages.json]: look at point 2

File: Extension/pages/skin/my.css [hunk 3]
```diff
@@ -273,6 +273,7 @@ a.spl-user-add-spn {
 }
 
 .popup-footer span {
```
Comment (ameshkov) [inline Extension/pages/skin/my.css]: One more thing:
```
.popup-footer {
    padding-left: 15px;
    padding-right: 15px;
}
```

File: Extension/_locales/en/messages.json (hunk 2 above)
Comment (Mizzick) [inline Extension/_locales/en/messages.json]: okay

**Pull request merged**